clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy

```diff
@@ -203,15 +205,48 @@ class UpsertAmazonLoadBalancerAtomicOperation implements AtomicOperation<UpsertA
         task.updateStatus BASE_PHASE, "Healthcheck configured."
       }
 
-      // Apply balancing opinions...
-      loadBalancing.modifyLoadBalancerAttributes(
-        new ModifyLoadBalancerAttributesRequest(loadBalancerName: loadBalancerName)
-          .withLoadBalancerAttributes(
-          new LoadBalancerAttributes(
-            crossZoneLoadBalancing: new CrossZoneLoadBalancing(enabled: description.crossZoneBalancing)
+      CrossZoneLoadBalancing crossZoneLoadBalancing = null
+      ConnectionDraining connectionDraining = null
+
+      if (loadBalancer) {
+        def currentAttributes = loadBalancing.describeLoadBalancerAttributes(new DescribeLoadBalancerAttributesRequest().withLoadBalancerName(loadBalancerName)).loadBalancerAttributes
```

Comment:
biggest change is to first do a describe attributes and only modify if a change has been explicitly requested